runtimes/pythonrt/pythonrt.go

```diff
@@ -402,7 +400,7 @@ func (py *pySvc) Close() {
 // We split it from Call since Call is also used to execute activities.
 func (py *pySvc) initialCall(ctx context.Context, funcName string, event map[string]any) (sdktypes.Value, error) {
 	defer func() {
-		py.log.Info("python done, killing")
+		py.log.Info("Python subprocess cleanup after initial call is done")
```

Comment:
Is "killing" implied then?

Reply:
Yes, but it's more than just killing the process, it's also disposing of the input/output streams communicating with it (lines 405-415).